xrtpy/response/xrt_teem.py

```diff
@@ -0,0 +1,397 @@
+import sys
+import numpy as np
+from astropy.constants import c, h
+from astropy import units as u
+from xrtpy.response.temperature_response import TemperatureResponseFundamental
+from sunpy.coordinates.sun import angular_radius
+
+
+def xrt_teem(hdr1, data1, hdr2, data2, binfac=1, Trange=None,
+             no_threshold=False, Te_err_threshold=0.5,
+             photon_noise_threshold=0.2, mask=None, verbose=False):
+    """
+    Get coronal temperatures and emission measures from a pair of images using
+    the filter ratio method.
+
+    Currently this program uses the solar spectrum calculated with CHIANTI
+    database ver. 6.0.1 (density: :math:`10^9` cm\ :sup:`-3`\ , ionization
+    equilibrium: chianti.ioneq, abundance: sun_coronal_ext), because this
+    is the only spectrum available in xrtpy. We expect this to change.
+
+    Parameters:
+    -----------
+    hdr1 : fits header
```

Comment:
```suggestion
    hdr1 : `~astropy.io.fits.HDUList`
```
If we link to a code object in the type specification, then we'll be able to link to the corresponding page in Astropy's docs through the magic of [intersphinx](https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html). 🪄

Feel free to edit this if this if I'm linking to the wrong code object.  (There's also a chance that it'd need to be truncated to `~astropy.io.HDUList` depending on how they have intersphinx set up.)

Reply:
What would you think of `xrt_teem` accepting `hdu1` and `hdu2` instead of `data1`, `hdr1`, `data2`, and `hdr2`?  Then inside the function, `data1` and `hdr1` could be extracted from `hdu1`, etc.

Reply:
I'd be fine with that. I asked about that at a meeting we had when I was starting the project. In a lot of ways it makes sense and is more compact. It differs more from the IDL code, but not so much as to make a big difference. Another option would be to provide filenames and/or file objects.
The only downside to providing hdus is that they should be closed after extracting the header and image. That can be done using a context manager, but then that's outside the scope of the function. And I don't think it's good practice (if even allowed) to close an hdu within a function that was opened outside of it.